src/blog/2023/02/community-news-02.md

```diff
@@ -17,8 +17,9 @@ If you've got something that you think we should share on our newsletters please
 
 ## News
 
-### [Node-RED Ask Me Anything](https://flowforge.com/ask-me-anything/ama-nodered/)
-[![AMA Session with Nick O'Leary and Rob Marcer](https://flowforge.com/images/webinars/ama-node-red.jpg)](https://flowforge.com/ask-me-anything/ama-nodered/)
+### [Node-RED Ask Me Anything](/ask-me-anything/ama-nodered/)
```

Comment:
@robmarcer It's better to use relative links, so local development doesn't link it to ff.com